anta/reporter/__init__.py

```diff
@@ -254,3 +264,73 @@ def render(self, data: list[dict[str, Any]], *, trim_blocks: bool = True, lstrip
             template = Template(file_.read(), trim_blocks=trim_blocks, lstrip_blocks=lstrip_blocks)
 
         return template.render({"data": data})
+
+
+class ReportCsv:
```

Comment:
Duplicate code? Since you moved it to a seperate module.

Reply:
Yes true. Code removed